.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+## Description
+
+
+## Screenshots / Videos (frontend only) 
+
+> Be sure to include both desktop and mobile views
```

Comment:
Should specify smallest mobile screen size is 320 pixels if possible. Some pages we could assume desktop view though.

Reply:
Added a note

I felt like it got too wordy when mentioning that some pages would only need desktop. I figure people can deviate if that's the case since it's just meant to be a template